### ISSUE TITLE
Admin logs page: Fix display of logs table on mobile

### ISSUE THREAD

**Issue by zhaojj2209** [TEAMMATES/teammates#12292]
Currently, the logs table fits within the window regardless of width, causing the table contents to be unreadable at small screen sizes:
![photo_2023-03-28 14 00 50](https://user-images.githubusercontent.com/54243224/228186289-d5453970-f135-437a-bb8d-ae9246bc4fe0.jpeg)

One possible fix is to make the table scroll horizontally at smaller screen sizes.

**Comment by ArslanYM** [comment]
Can i work on this ?

**Comment by domlimm** [comment]
@ArslanYM Hello, thanks for your interest in contributing! Please only select 1 issue of the 3 that you commented on. Issues tagged with `good first issue` are for first time contributers and contributors should only work on a maximum of one of these issues.

Comment again on which one of the three you would like to get started on. Thanks!

**Comment by ArslanYM** [comment]
I would like to work on this one

**Comment by mud1tx** [comment]
@zhaojj2209 I can work on this issue.

**Comment by Anshul-target** [comment]
Hello sir Can I work on this
Or directly create a PR

**Comment by cedricongjh** [comment]
Hi @mud1tx and @Anshul-target, 

@ArslanYM has expressed interest in resolving the issue. Let's check if he's still working on it first.

@ArslanYM, do let us know if you're still working on this issue, thank you!

**Comment by ArslanYM** [comment]
No I'm not. You can assign someone else. Sorry for the inconvenience

**Comment by cedricongjh** [comment]
Hi @ArslanYM, thank you for your quick response!

@mud1tx, I see that you're currently working on https://github.com/TEAMMATES/teammates/issues/12279, thus @Anshul-target, feel free to work on this issue!

**Comment by Anshul-target** [comment]
Hello sir I got this issue while setting the project my local system 


![Problem](https://user-images.githubusercontent.com/118791090/233174684-be0d4467-e5a7-47e6-b9d2-8e4e7169c65c.png)

But Sir I have just set it in the environment variable but it still giving me the same error


![Screenshot 2023-04-20 002920](https://user-images.githubusercontent.com/118791090/233174973-f125e519-257d-4d65-bb0e-428e6f38ed89.png)

**Comment by cedricongjh** [comment]
> Hello sir I got this issue while setting the project my local system
> 
> ![Problem](https://user-images.githubusercontent.com/118791090/233174684-be0d4467-e5a7-47e6-b9d2-8e4e7169c65c.png)
> 
> But Sir I have just set it in the environment variable but it still giving me the same error
> 
> ![Screenshot 2023-04-20 002920](https://user-images.githubusercontent.com/118791090/233174973-f125e519-257d-4d65-bb0e-428e6f38ed89.png)

Hi @Anshul-target, do try to close and open your terminal again after setting the environment variable. You can run the `set java_home` command in your terminal to check if the environment variable has been set, it should output `JAVA_HOME=C:\Program Files\Java\jdk-11`

**Comment by Anshul-target** [comment]
Oh it worked after I restarted my PC 😅

**Comment by Anshul-target** [comment]
Sir please help 
When i run npm run start 
![config](https://user-images.githubusercontent.com/118791090/233190105-d9a78332-733a-4df0-8af1-2fa4b2b7f0cd.png)

**Comment by weiquu** [comment]
Hi @Anshul-target, let's shift this to a discussion thread since it's not relevant to the issue. While you're opening the discussion, do also make sure that you've followed all the steps as detailed in our [documentation](https://teammates.github.io/teammates/setting-up.html).

**Comment by Anshul-target** [comment]
@zhaojj2209 would you please tell me how  could I reach to this issue In the Teammates

**Comment by zhaojj2209** [comment]
@Anshul-target As mentioned in the issue title, the issue can be found in the admin logs page.

**Comment by Anshul-target** [comment]
@cedricongjh Sir any changes in the frontEnd is not upading
Please Help

**Comment by zhaojj2209** [comment]
@Anshul-target Note that you should only pick one issue with the `good first issue` label to work on. Since you have commented on another issue with the `good first issue` label, can we confirm that you are no longer working on this issue?

**Comment by Anshul-target** [comment]
@zhaojj2209 Mam I was trying to fix this for hours but any changes made on the .scss files were not get updated in my local setup of TEAMATES  so I moved to another issue

**Comment by zhaojj2209** [comment]
@Anshul-target What you are describing is still too vague, the only advice I can give is to read up on Bootstrap and SCSS, as well as explore the codebase to better understand how we style our components. This knowledge is necessary if you wish to contribute to the TEAMMATES frontend, without it you're likely to run into the same problems even if you try your luck with other issues.

**Comment by yash29896** [comment]
Is anyone actively working on this ? Id like to pick this up as my first issue.

**Comment by domlimm** [comment]
@yash29896 Hey! Feel free to submit a PR for this issue.

**Comment by yash29896** [comment]
Following UT is failing 
TEST FAILURE:  [https://github.com/TEAMMATES/teammates/actions/runs/4839710230/jobs/8624902930?pr=12411](url)

Seems like the component isnt matching the existing snapshot. 

How can I update the existing snapshot for the logs table page ? 
I followed the documentation which led me to this property test.snapshot.update=false in test.properties.

_Steps Followed_

1. Changed the UT from it to fit
2. set test.snapshot.update=true
3. ran `npm run test`
4. still got the same failure
5. set test.snapshot.update=true
6. ran `npm run test`
7. got the same failure again
